### ISSUE TITLE
SignShop link process not working

### ISSUE THREAD

**Issue by rcnsol** [AlmuraDev/Almura#232]
The link process is not working for me.
Order of actions:

1. Place chest (vanilla) 
2. Insert item (1 Ender Pearl)
3. Place sign (the contained text is in correct format)
4. Left click chest: [SignShop] Stored location of Chest Facing South containing 1 Ender Pearl
5. Left click sign: [SignShop] Stored location of Wall Sign Facing South

The link process is not completed. It just stays at that.

(robitect)

**Comment by Wolfeyeamd0** [comment]
So after some testing wit robitect we found out a few things.  

1.  It doesn't seem to be related to the res.  I took him out in the middle of nowhere it still didn't work.

2. If I write the sign and he connects it, the sign shop links.  If he writes the sign it doesn't. 

* I made sure we were both writing the sign the same way so it's not that.

**Comment by rcnsol** [comment]
I tried repeating what Wolfeyeamd0 and I tried with tinacoladas account, but it did not work. We are using the same internet connection however. 

What could help would be to check the sellers.yml in /plugins/SignShop/ and erase all entries corresponding with my UUID (i would provide that by PM) and the same for the entry in /plugins/SignShop/db/player.db, if there is a SQLite editor at hand.

I am sorry to cause inconveniences, i really do not know how this could have happened.
